ruma-common/src/encryption.rs

```diff
@@ -0,0 +1,37 @@
+//! Common types for encryption related tasks.
```

Comment:
This should probably point to the client-server APIs end-to-end encryption module docs, similar to the other module's doc comment.s